src/helpers/similarity.ts

```diff
@@ -0,0 +1,106 @@
+import { getLogger } from "../bindings";
+import axios, { AxiosError } from "axios";
+import { ajv } from "../utils";
+import { Static, Type } from "@sinclair/typebox";
+import { backOff } from "exponential-backoff";
+
+export const extractImportantWords = async (content: string): Promise<string[]> => {
```

Comment:
```suggestion
export const extractImportantWords = async (issue: Issue): Promise<string[]> => {
```

Let the function extract what it needs